pkg/storaged/iscsi-panel.jsx

```diff
@@ -91,16 +89,14 @@ export class IscsiPanel extends React.Component {
                                             address: _("Unable to reach server")
                                         };
 
-                                    dfd.reject(error);
+                                    reject(error);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18487-20230310-143650-f841503b-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)